docs/dictionary/command/create-widget.lcdoc

```diff
@@ -0,0 +1,44 @@
+Name: create widget
+
+Type: command
+
+Associations: widget
+
+Syntax: create [invisible] widget [<controlName>] as <widgetKind> [in <group>]
+
+Summary: Create a <widget> control of the specified <kind>.
```

Comment:
Throughout this documentation block, I think you need `<kind>` -> `<widgetKind>` to match the syntax summary...